bootstrap/config/config_test.go

```diff
@@ -92,3 +108,237 @@ func TestGetSecretPathsChanged(t *testing.T) {
 		})
 	}
 }
+
+func TestLoadCommonConfig(t *testing.T) {
+	serviceKey := "test-service"
+	getAccessToken := func() (string, error) {
+		return "", nil
+	}
+	serviceConfig := ConfigurationMockStruct{
+		Writable: WritableInfo{
+			LogLevel: "INFO",
+		},
+		Registry: config.RegistryInfo{
+			Host: "localhost",
+			Port: 8500,
+			Type: "consul",
+		},
+	}
+
+	appConfig := ConfigurationMockStruct{
+		Writable: WritableInfo{
+			StoreAndForward: StoreAndForwardInfo{
+				Enabled:       false,
+				RetryInterval: "5m",
+				MaxRetryCount: 10,
+			},
+		},
+		Trigger: TriggerInfo{
+			Type: "edgex-messagebus",
+		},
+	}
+
+	deviceConfig := ConfigurationMockStruct{
+		Writable: WritableInfo{
+			Telemetry: config.TelemetryInfo{
+				Metrics: map[string]bool{"EventsSent": false, "ReadingsSent": true},
+			},
+		},
+	}
```

Comment:
Please update the mock struct to only have the sections you are using here.

Reply:
done